Website link updated.

BihevRawal opened the pull request:
<!--
YOU MUST FILL OUT THIS TEMPLATE ENTIRELY FOR YOUR PR TO BE ACCEPTED, NONE OF IT IS OPTIONAL UNLESS SPECIFIED.
-->

# Requirements
<!-- Your domain MUST pass ALL the requirements below, otherwise it WILL BE DENIED. -->

<!-- Change each checkbox to [x] (all lowercase, with no spaces between the brackets) to mark it as completed. -->

- [ X] I have **read** and **agreed** to the [Terms of Service](https://is-a.dev/terms). <!-- Your request MUST follow the TOS to be approved. -->
- [ X] My file is following the [domain structure](https://docs.is-a.dev/domain-structure/).
- [ X] My website is **reachable** and **completed**. <!-- We do not permit simple "Hello, world!" or simply copied/mostly blank templated websites. -->
- [ X] My website is **software development** related. <!-- Only your root subdomain needs to meet this requirement. -->
- [ X] My website is not for commercial use. <!-- Your website's purpose should not be to generate any form of revenue. -->
- [ X] I have provided sufficient contact information in the `owner` key. <!-- Provide your email in the `email` field or another platform (e.g. X/Twitter or Discord) for contact. -->
- [ X] I have provided a preview of my website below. <!-- This step is required for your domain to be approved. -->

# Website Preview
https://bihev-portfolio-client.vercel.app/
<!-- Provide a link or screenshot of your website below. -->
<img width="1463" height="739" alt="Screenshot 2025-09-25 at 3 15 28 am" src="https://github.com/user-attachments/assets/3324f914-7828-4354-b719-e6e3167fd24a" />

BihevRawal committed:
Create bihevportfolio.json

add bihev.is-a.dev subdomain

BihevRawal committed:
Update bihevportfolio.json

subdomanin changed to bihev from bihevportfolio

BihevRawal committed:
Update bihevportfolio.json

correct CNAME update

iostpa commented on domains/bihevportfolio.json:
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Bihev Singh Rawal's Portfolio",
+  "subdomain": "bihev",
```
```suggestion
```

iostpa commented on domains/bihevportfolio.json:
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Bihev Singh Rawal's Portfolio",
+  "subdomain": "bihev",
+  "owner": {
+    "repo": "https://github.com/BihevRawal/bihev-portfolio-client",
+    "email": "bihevr@gmail.com"
+  },
```
You need to add your GitHub username here. Check the documentation at the "Domain structure" part.

iostpa commented on domains/bihevportfolio.json:
```diff
@@ -0,0 +1,11 @@
+    "repo": "https://github.com/BihevRawal/bihev-portfolio-client",
+    "email": "bihevr@gmail.com"
+  },
+  "record": {
+    "CNAME": "https://bihev-portfolio-client.vercel.app/"
+  }
```
```suggestion
  "records": {
    "CNAME": "bihev-portfolio-client.vercel.app"
  }
```

iostpa reviewed (dismissed):
You would also need to verify your domain with Vercel. To save the hassle do it in this pull request, for the file format check our documentation.

BihevRawal committed:
Create bihev.json

fix: correct filename, owner.username, records, and Vercel TXT verification

BihevRawal committed:
Delete domains/bihevportfolio.json

chore: remove duplicate JSON file

STICKnoLOGIC commented on domains/bihev.json:
```diff
@@ -0,0 +1,13 @@
+    "repo": "https://github.com/BihevRawal/bihev-portfolio-client"
+  },
+  "records": {
+    "CNAME": "bihev-portfolio-client.vercel.app",
+    "TXT": "vc-domain-verify=bihev.is-a.dev,6f831b0c66c44576f576"
```
check the step 5 -> https://docs.is-a.dev/guides/vercel/


```suggestion
    "CNAME": "cname.vercel-dns.com"
```

BihevRawal committed:
Update bihev.json

Updated per docs — CNAME now points to cname.vercel-dns.com only.

BihevRawal committed:
Update bihev.json

Updated as requested: removed owner.repo. Records use CNAME=cname.vercel-dns.com.

STICKnoLOGIC commented on domains/bihev.json:
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Bihev Singh Rawal's Portfolio",
+  "subdomain": "bihev",
```
```suggestion
```

BihevRawal committed:
Update bihev.json

fix: remove redundant subdomain field

STICKnoLOGIC reviewed (approved):
lgtm, dont forget to finish step 5 -> https://docs.is-a.dev/guides/vercel/
